packages/backend/src/index.ts

```diff
@@ -672,13 +672,16 @@ export function initIocContainer(
     })
   })
 
-  container.singleton('cardService', async (deps) => {
-    return createCardService({
-      axios: await deps.use('axios'),
-      logger: await deps.use('logger'),
-      cardServiceUrl: config.cardServiceUrl
+  if (config.cardServiceUrl) {
```

Comment:
Let's remove this `if` and keep the `cardService` service defined, otherwise we might get unexpected behaviour if a singleton resolves to undefined.